### PR TITLE
add customerType tag

### PR DESCRIPTION
**Pull request by sdzhong**
resolves https://github.com/sentry-demos/android/issues/129

### CODE REVIEW

File: app/src/main/java/com/example/vu/android/MyApplication.java [hunk 1]
```diff
@@ -111,6 +111,9 @@ public void onCreate() {
             });
         });
 
+        String[] allCustomerTypes = {"medium-plan", "large-plan", "small-plan", "enterprise"};
+        String customerType = allCustomerTypes[(int) (Math.random() * 4)];
+        Sentry.setTag("customerType", customerType);
```
Comment (sdzhong) [inline app/src/main/java/com/example/vu/android/MyApplication.java:116]: customerType added to SentryAndroid.init()

Comment (sdzhong) [reply app/src/main/java/com/example/vu/android/MyApplication.java:116]: image

Comment (sdzhong) [reply app/src/main/java/com/example/vu/android/MyApplication.java:116]: image

Comment (sdzhong) [reply app/src/main/java/com/example/vu/android/MyApplication.java:116]: Note: other events also have their associated customerType
image

**Review by realkosty — APPROVED** [review]
nice

**Pull request merged**